charts/catalog/templates/controller-manager-deployment.yaml

```diff
@@ -54,6 +54,7 @@ spec:
         - --service-catalog-api-server-url
         - https://{{ template "fullname" . }}-apiserver
         {{- end }}
+        - "--service-catalog-insecure-skip-verify=true"
```

Comment:
Why on by default? I feel like that should be mentioned in the readme/walkthrough somewhere. I'd prefer if we could have everything be secure by default. And if not, I'd like to fully understand why.